src/Orleans.Membership.Azure.TableStorage/Orleans.Membership.Azure.TableStorage.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Membership.Azure.TableStorage</PackageId>
+    <Title>Microsoft Orleans Membership Azure Table Storage</Title>
+    <Description>Membership provider for Azure Table Storage of Microsoft Orleans.</Description>
+    <PackageTags>$(PackageTags) Azure</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>Orleans.Membership.Azure.TableStorage</AssemblyName>
```

Comment:
Was this the final decision on naming? I thought it was `Orleans.Membership.AzureTable`, and then we can have `Orleans.Membership.CosmosDb` and what not.
I'm not pushing back, and fine with whatever you guys decide, is just that I remember differently. /cc @sergeybykov

Reply:
I like `Microsoft.Orleans.Membership.AzureTable` better than `Microsoft.Orleans.Membership.Azure.TableStorage`. Although Microsoft.Orleans.ClusterMembership.AzureTable` seems more clear to me.

Reply:
During the meeting, we decided that `Clustering` is a clearer name than `Membership` for most users

Reply:
We also decided we would call packages `AzureStorage`. instead of `AzureTables`, etc

Reply:
For package names, we decided to use `Persistence` instead of `Storage`, like `Microsoft.Orleans.Persistence.AzureStorage`